exercises/sum-of-multiples/instructions.md

```diff
@@ -0,0 +1,33 @@
+# Instructions
+
+Your task is to write the code that calculates the energy points that get awarded to players when they complete a level.
+
+The points awarded depend on two things:
+
+- the number of the level that the player completed
+- the base value of each magical item collected by the player during that level
+
+The energy points are awarded according to the following rules:
+
+- For each magical item, take the base value and find all the multiples of that value that are less than or equal to the level number.
+- Combine the sets of numbers.
+- Remove any duplicates.
+- Calculate the sum of all the numbers that are left.
+
+For example, let's say that the player completed level 20 and found two magical items with base values of 3 and 5.
+
+To calculate the energy points earned by the player, we need to find all the unique multiples of these base values that are less than or equal to level 20.
+
+For the base value of 3, the multiples less than or equal to 20 are: 3, 6, 9, 12, 15, 18, and 20.
+
+For the base value of 5, the multiples less than or equal to 20 are: 5, 10, 15, and 20.
+
+To find the unique multiples, we can combine these two sets of multiples and remove any duplicates:
+
+{3, 5, 6, 9, 10, 12, 15, 18, 20}
+
+The sum of all these unique multiples is:
+
+3 + 5 + 6 + 9 + 10 + 12 + 15 + 18 + 20 = 98
+
+Therefore, the player earns 98 energy points for completing level 20 and finding the two magical items with base values of 3 and 5.
```

Comment:
```suggestion
Let's look an example;

- The player completed level 20 and found two magical items with base values of 3 and 5.
- To calculate the energy points earned by the player, we need to find all the unique multiples of these base values that are less than or equal to level 20.
- For the base value of 3, the multiples less than or equal to 20 are: 3, 6, 9, 12, 15, 18, and 20.
- For the base value of 5, the multiples less than or equal to 20 are: 5, 10, 15, and 20.
- To find the unique multiples, we can combine these two sets of multiples and remove any duplicates: `{3, 5, 6, 9, 10, 12, 15, 18, 20}`
- The sum of all these unique multiples is: `3 + 5 + 6 + 9 + 10 + 12 + 15 + 18 + 20 = 98`
- Therefore, the player earns **98** energy points for completing level 20 and finding the two magical items with base values of 3 and 5.
```

This will shorten it a lot when rendered, plus I think it's more comprehensable.